src/Numerics/project.json

```diff
@@ -0,0 +1,97 @@
+﻿{
+  "name": "MathNet.Numerics.Core",
+  "title": "Math.NET Numerics",
+  "description": "Math.NET Numerics is the numerical foundation of the Math.NET project, aiming to provide methods and algorithms for numerical computations in science, engineering and every day use.",
+  "authors": [ "Christoph Ruegg", "Marcus Cuda", "Jurgen Van Gael" ],
+
+  "version": "3.17.0",
+  "dependencies": {
+
+  },
+
+  "frameworks": {
+    "netstandard1.0": {
+      "imports": "dnxcore50",
```

Comment:
Why would .Net Standard import .Net Core? Isn't the `dnx` stuff long deprecated? 😕